packages/expo-task-manager/android/src/main/java/expo/modules/taskManager/TaskManagerModule.kt

```
@@ -30,6 +31,25 @@ class TaskManagerModule : Module() {
       "EVENT_NAME" to TaskManagerInterface.EVENT_NAME
     )
 
+    OnCreate {
+      // Slightly hacky way to be able to emit events using the new event emitter from legacy modules.
+      val weakModule = WeakReference(this@TaskManagerModule)
+      val emitEvent = { name: String, body: Bundle ->
+        try {
+          // It may throw, because RN event emitter may not be available
+          // we can just ignore those cases
+          weakModule.get()?.sendEvent(name, body)
+        } catch (_: Throwable) {
+        }
```

Let's log that error. It may be hard to debug that if we suppress that error.